src/consensus/params.h

```diff
@@ -11,13 +11,25 @@
 
 namespace Consensus {
 
-enum DeploymentPos
+enum BuriedDeployment : int16_t
+{
+    // buried deployments get negative values to avoid overlap with SignalledDeployment
```

Comment:
SignalledDeployment == DeploymentPos? 

although, I'd probably be down for a rename. I've been wondering what Pos stands for? position?

Reply:
"position in the vDeployments" array, I presume. Yeah, I think I changed it to "SignalledDeployment" at some point then dropped it as being too annoying to keep rebasing